src/overlays/actors/ovl_En_Toryo/z_en_toryo.c

```diff
@@ -281,64 +281,64 @@ s32 func_80B206A0(EnToryo* this, PlayState* play) {
     return ret;
 }
 
-void func_80B20768(EnToryo* this, PlayState* play) {
+void EnToryo_Talk(EnToryo* this, PlayState* play) {
     Player* player = GET_PLAYER(play);
```

Comment:
I expressed this in my previous comment, but may have not been clear, but I dont think this one should be named a simple verb like an action function when it isnt really one. Id be fine with `HandleTalking` or something